### ISSUE TITLE
is there a plan to update the docker image with glue version 4.0 

### ISSUE THREAD

**Issue by mpulaparthi** [awslabs/aws-glue-libs#165]

**Comment by singlewind** [comment]
Yes, I think so. I'm waiting the image to be released as well so I can update in pipeline. I suggest release the Dockerfile repo as well, so we can update by yourself. https://github.com/awslabs/aws-glue-libs/issues/158#issuecomment-1353889589

**Comment by dmschauer** [comment]
Indeed, please release the dockerfile. It would help lots of people with their development workflow, see: 
https://github.com/awslabs/aws-glue-libs/issues/81

**Comment by PykupeJIbc** [comment]
I join the previous speakers!

**Comment by michal-ziarnik** [comment]
upvoting request

**Comment by saviodsouza29** [comment]
This issue has been resolved. - https://hub.docker.com/r/amazon/aws-glue-libs/tags?page=1&name=4.0.0

**Comment by dmschauer** [comment]
@saviodsouza29 What about the Dockerfile?

**Comment by singlewind** [comment]
Can we rebase the container to Amazon Linux 2022 instead of Amazon Linux 2?

**Comment by singlewind** [comment]
The reason why I ask this is by default the python version on the container is 3.10.2 and glibc 2.26 which is OK. If we would like to install a newer version like 3.10.9 which depends on glibc 2.34. Amazon linux 2022 has glibc 2.34 already.
